.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,6 @@
 ## Pull Request checklist:
 
-- [ ] Write an explicit title for the Pull Request
-- [ ] Write changes made in the CHANGELOG.md
+- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
```

Comment:
As the the git hook will prevent bad formatting, shouldn't we simply remove this line too?

Reply:
I think it's worth adding a link to the convention for now (for learning purpose), and maybe remove it later. Also, the developer has to remember to _name_ her commit because she will have to choose one (and review it), even if there is a bunch of commits.

Not sure though: tell me what you prefer!

Reply:
Let's keep it like that!